package.json

```diff
@@ -48,13 +48,12 @@
     "@ember/test-helpers": "^2.6.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
-    "@kanselarij-vlaanderen/au-kaleidos-css": "2.5.1",
+    "@kanselarij-vlaanderen/au-kaleidos-css": "2.5.2",
     "@kanselarij-vlaanderen/au-kaleidos-icons": "3.0.0",
     "@lblod/ember-acmidm-login": "~1.3.0",
     "@lblod/ember-mock-login": "^0.5.0-beta",
     "@lblod/ember-rdfa-editor": "0.50.0-beta.4",
     "@lblod/ember-rdfa-editor-plugin-system-dispatcher": "^0.5.8",
-    "@lblod/ember-vo-webuniversum": "^0.23.0",
```

Comment:
@MikiDi Voor het verwijderen van de overbodige CSS/styling was ik eerst ten rade gegaan bij Dieter. Uiteindelijk heb ik het op dezelfde manier aangepakt zoals zij dat hebben gedaan:
- `ember-vo-webuniversum` package uit het project halen.
- De (nog) nodige componenten migreren (in ons geval enkel `wu-icon` omdat `ember-vo-webuniversum-data-table` die nodig heeft).
- Daarnaast ook de (nog) nodige CSS migreren en filteren waar nodig.

Van zodra dit afgerond is zal ik ook eens kort een item opnemen om:
- Alle legacy tabellen (in de vorm van `ember-vo-webuniversum-data-table`) te doorlopen met het oog om die allemaal te vervangen/refactoren.
- De resterende CSS + selectors binnen de wu-namespace uit de app te halen.

Reply:
@MikiDi FYI: Uiteindelijk is dat op vlak van CSS/styling een winst van een goeie 200kb bij de first load. Mbt performantie (zowel build als render tijd mbt een hoop minder CSS selectors) zal ook het wel wat impact hebben. 🥳